### ISSUE TITLE
DCML>RNTXT conversion (bug). 4x fail to load.

### ISSUE THREAD

**Issue by MarkGotham** [MarkGotham/When-in-Rome#81]
## DCML Missing scores

First, the new DCML repos are missing some scores (reported there). Note to self: update when they fill in the gaps.

## Conversion from DCML fails

Hi @malcolmsailor . I ran the DMCL-rntxt converter on all the new DCML analyses and have mostly great results. Some exceptions:

Parses and writes fine, but probably warrants a clean up:
- Some `[no5][no5]` duplication in DCML to rntxt. E.g., `Corpus/Keyboard_Other/Chopin,_Frédéric/Mazurkas/BI145-1/analysis.txt`

Actually fails to parse = 4 cases in total:
- `Corpus/Keyboard_Other/Liszt,_Franz/Années_de_pèlerinage/S161_5_Sonetto_104_del_Petrarca/`
- `Corpus/Keyboard_Other/Liszt,_Franz/Années_de_pèlerinage/S160_6_Vallee_dObermann/`
- `Corpus/Keyboard_Other/Medtner,_Nikolai/Tales/Op34_No3/`
- `Corpus/Keyboard_Other/Medtner,_Nikolai/Tales/Op35_No2/`

Example error message:
- `Corpus/Keyboard_Other/Liszt,_Franz/Années_de_pèlerinage/S161_5_Sonetto_104_del_Petrarca` raises this exception `music21.roman.RomanNumeralException: Cannot make a dominant-seventh chord out of 7[addb1o]. Figure should be in ('7', '65', '43', '42', '2')`

**Comment by malcolmsailor** [comment]
Glad to hear the DCML converter is proving useful!

In the example you gave, it looks like the parser is getting confused by either or both of the "b" and "o" annotations. Presumably, "o" should apply to the chord and not to the 1 figure.

If you want to see if you can patch up the parsing so that it can fix these cases, I have a notebook for comparing DCML and music21 parsings that Johannes made and I further altered. I will send it to you by email.

**Comment by malcolmsailor** [comment]
The merged PRs https://github.com/cuthbertLab/music21/pull/1635 and https://github.com/cuthbertLab/music21/pull/1634 address two of the 3 bugs causing these issues (see those PRs for more details). The last one is a bug in ms3 that produces negative `mc_onset` values. @johentsch (not sure why tagging him doesn't seem to be working) said he will fix that.

**Comment by johentsch** [comment]
Yes tagging works, and the bugs have been fixed (upgrade to the latest `ms3`). Now, when negative `mc_onset` values occur it's (most probably) because of an error in the encoding, in which case one needs to find the label (find out where it had correctly been placed) and re-insert it in the correct position. Needless to say, we gladly accept new issues if it occurs in our corpora.

**Comment by malcolmsailor** [comment]
> Yes tagging works

Oh weird... for some reason it wasn't giving me the usual preview that shows that an individual has been tagged. Glad it worked! And thanks for the fix!
